src/test/java/com/business/entities/Admin_getAdminEmail_d02ae8178d_Test.java

```diff
@@ -27,58 +27,50 @@
 */
 package com.business.entities;
 
-import org.junit.Assert;
-import org.junit.Test;
-import org.springframework.beans.factory.annotation.Value;
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import jakarta.validation.constraints.Email;
-import jakarta.validation.constraints.NotNull;
-import jakarta.validation.constraints.Size;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 
 public class Admin_getAdminEmail_d02ae8178d_Test {
 
     @Test
     public void testGetAdminEmail() {
         Admin admin = new Admin();
         admin.setAdminEmail("admin@example.com");
-        Assert.assertEquals("admin@example.com", admin.getAdminEmail());
```

Comment:
Fix : Assert.assertEquals is often associated with JUnit 4, while Assertions.assertEquals is associated with JUnit 5. If your project is configured to use JUnit 5, you should consistently use Assertions.assertEquals.